web/app/scripts/src/directives/proportion-field.es.js

```diff
@@ -66,6 +66,16 @@ module.directive('proportionField', [
                         return;
                     }
 
+                    if ( Object.values(newModel).every( (x) => { return x === null; } ) ) {
```

Comment:
You don't need the curleys `{}` or the return if there is only one command `Object.values(newModel).every((x) => x === null)`